### DISCUSSION TITLE
Welcome to ambianic-box Discussions!

### DISCUSSION THREAD

**Discussion by ivelin** [ambianic/ambianic-box#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nueladeyemi1** [comment]
The initial prototype is cool but have some limitations
<li>Once coupled Micro SD card will not be able to be removed unless it is uncoupled 
<li>2 x USB2.0, 2 x USB3.0 and Gigabit Ethernet ports will be embedded in the casing once coupled
<li> 4 pole stereo audio and USB C are also embedded.

Though the limitation do not affect the functionality but I am sharing this version of casing as an improvement over the existing casing.

This new version has
<li>Good aesthetic and it will be easy to couple with 4 x M2.75 screws
<li> Micro SD card space is visible for easy remove and fixing
<li> 2 x USB2.0, 2 x USB3.0 and Gigabit Ethernet ports are visible for working even after coupling
<li> 4 pole stereo audio and USB C are visible for working even after coupling
<li> It has an emboss of the company's name on the front side of the casing, the emboss is of 0.5mm height
<li> Like the prototype version, it has a groove for hanging upright and four (4) short stable stand around the base to make it stand rigidly upright on a surface
<li> Aside being for aesthetic, the engraving of the company's logo on the casing is for aeration against heat build in the system (should there is one)
<li> It is has a very small space around the top to push finger into for easy decoupling

The casing can be access, view in different positions and downloaded with this Autodesk public link<a href='https://a360.co/3kDUyTY' > **HERE**</a>

Here are some pictures of this version of the casing
![Ambiamic_BoxCasing](https://user-images.githubusercontent.com/91387021/134786158-52974013-52ab-446d-91f4-7d142ebfdc64.jpg)
![Ambiamic_BoxCasing2](https://user-images.githubusercontent.com/91387021/134786163-39cd2e91-a448-4664-9e46-7ff4aa9cce2c.jpg)
 
![Ambiamic_BoxCasing3](https://user-images.githubusercontent.com/91387021/134786165-c86d7a6b-a782-49f5-ba92-1ea47759ea02.jpg)
![Ambiamic_BoxCasing4](https://user-images.githubusercontent.com/91387021/134786168-7f15f15b-478f-49d0-b82b-0f565e88a071.jpg)
![Screenshot (282)](https://user-images.githubusercontent.com/91387021/134786171-c787c9e4-ad9c-4847-a66f-d864389929fc.png)


If you notice anything to improve on this existing casing, do join the discussion. Thanks

**Reply by ivelin** [reply]
Nice work. I like this as a potential hacker version of the ambianic box. Have you 3D printed it yet? Photos?

The basic version assumes that the box will be assembled once, mounted somewhere and left alone for weeks or months. Like a regular end user camera. With automatic over-the-air software updates. It was not intended to be used for regular hacking, which is why the ports were covered to prevent dust from damaging the device.
